_layouts/product.html

```diff
@@ -183,6 +183,18 @@
   <td {% if diff <= 0 %} class = "txt-linethrough" {% endif %} >      <!-- if the support finished add txt-linethrough class -->
     {% if releaseLink != "" %}
       <a href="{{releaseLink}}" title="Release Notes / Changelog">{{latestVersionNumber}}</a>
+{% if  diff > 1 %}
+      &ensp;
+      <a href="http://www.google.com/calendar/render?action=TEMPLATE
+&text=[endoflife.date] {{page.title}} {{r.releaseCycle}} EOL reached
+&dates={{r.eol | replace:'-',''}}T000000/{{r.eol | replace:'-',''}}T000000
+&details=It will be end of support for  {{page.title}} {{r.releaseCycle}}
+&trp=false
+&sprop=
```

Comment:
What are `trp` and `sprop` ? A link to a documentation explaining how to create a Google Calendar Button will be great as a comment in the template or in the commit message.

Reply:
the problem begins with it :D because they dont document anything i have found related documentation from time machine
https://web.archive.org/web/20120102120045/http://www.google.com:80/googlecalendar/event_publisher_guide.html#individual

Reply:
but i know what is trp . it is for repeateable reminders which we dont need

Reply:
Better than nothing. Do not forget to add this link in your commit message.